tracing/errors.go

```diff
@@ -1,9 +1,14 @@
 package tracing
 
 import (
+	"errors"
 	"fmt"
 )
 
+// ErrNotSpanHook is the error logged when you pass an object to span.AddHooks
+// that does not conform to any of the span hook interfaces.
+var ErrNotSpanHook = errors.New("Attempting to add non-SpanHook object into Span's hook registry")
```

Comment:
🔕 This error is only used in logging and never returned to caller, so does not really need to be exported.

style nit: don't capitalize error messages (golint or govet is supposed to complain about that, not sure why it did not happen).